spec/arns_spec.lua

```diff
@@ -787,39 +770,6 @@ describe("arns", function()
 					undernameLimit = 10,
 				},
 			}, _G.NameRegistry.records)
-			-- grace period record now joins the auction list and is pruned
-			assert.are.same({
-				["expired-record"] = {
-					startTimestamp = 2000000000,
-					endTimestamp = 3209600000, -- plus 14 days
-					initiator = _G.ao.id,
-					baseFee = 400000000,
-					demandFactor = 1,
-					registrationFeeCalculator = arns.calculateRegistrationFee,
-					name = "expired-record",
-					settings = {
-						decayRate = 0.02037911 / (1000 * 60 * 60 * 24 * 14),
-						scalingExponent = 190,
-						startPriceMultiplier = 50,
-						durationMs = 60 * 1000 * 60 * 24 * 14,
-					},
-				},
-				["grace-period-record"] = {
-					baseFee = 400000000,
-					demandFactor = 1.0,
-					startTimestamp = currentTimestamp,
-					endTimestamp = currentTimestamp + 1209600000, -- plus 14 days
-					initiator = "test",
-					name = "grace-period-record",
-					registrationFeeCalculator = arns.calculateRegistrationFee,
-					settings = {
-						decayRate = 0.02037911 / (1000 * 60 * 60 * 24 * 14),
-						scalingExponent = 190,
-						startPriceMultiplier = 50,
-						durationMs = 60 * 1000 * 60 * 24 * 14,
-					},
-				},
-			}, _G.NameRegistry.auctions)
 			assert.are.same({
```

Comment:
prune is now responsible for creating auctions and removing any primary names associated to arns names that are removed